nix/modules/fedimintd.nix

```diff
@@ -133,18 +133,25 @@ let
             example = "bitcoin";
             description = "Bitcoin network to participate in.";
           };
+
           bitcoindUrl = mkOption {
             type = types.nullOr types.str;
             default = null;
             example = "http://127.0.0.1:38332";
             description = "Bitcoin node (bitcoind/electrum/esplora) address to connect to";
           };
 
-          esploraUrl = mkOption {
+          bitcoindUser = mkOption {
             type = types.nullOr types.str;
```

Comment:
I guess we can make this

```
            type = types.str;
```

to force the users to fix it, and discover that the convention changed?